scrapli/driver/base/base_driver.py

```diff
@@ -613,16 +618,19 @@ def _resolve_ssh_config(self, ssh_config_file: str) -> str:
             N/A
 
         """
-        self.logger.debug("attempting to resolve 'ssh_config_file' file")
+        self.logger.debug(f"attempting to resolve 'ssh_config_file' file {ssh_config_file}")
 
         resolved_ssh_config_file = ""
 
-        if Path(ssh_config_file).is_file():
-            resolved_ssh_config_file = str(Path(ssh_config_file))
-        elif Path("~/.ssh/config").expanduser().is_file():
-            resolved_ssh_config_file = str(Path("~/.ssh/config").expanduser())
-        elif Path("/etc/ssh/ssh_config").is_file():
-            resolved_ssh_config_file = str(Path("/etc/ssh/ssh_config"))
+        if ssh_config_file == "" and transport == "system":
+            resolved_ssh_config_file = BaseTransport.SSH_SYSTEM_CONFIG_MAGIC_STRING
+        else:
```

Comment:
haven't looked too closely yet, but can we maybe "flatten" this out a bit if possible? guard clause or something -- like if this thing return, then no extra indented else block (hopefully that makes sense)

Reply:
Good point, flat is indeed better, should be better now!